tests/functional/aws-node-sdk/test/object/unsupportedQueries.js

```diff
@@ -0,0 +1,39 @@
+import assert from 'assert';
+import makeRequest from '../../lib/utility/makeRequest';
+
+// Queries supported by AWS that we do not currently support.
+const unsupportedQueries = [
```

Comment:
agree. easy to import from routeUtils or this could be moved to constants.js and we can update it as we support more.